src/runtime/lpython_builtin.py

```diff
@@ -806,6 +806,24 @@ def _lpython_str_endswith(s: str, suffix: str) -> bool:
         
     return True
 
+@overload
+def _lpython_str_partition(s:str, sep: str) -> tuple[str, str, str]:
+    """
+    Returns a 3-tuple splitted around seperator
+    """
+    if(len(s) == 0):
```

Comment:
```suggestion
    if len(s) == 0:
```

Reply:
Why are these changes required? Does it make some performance difference or is it some coding standard ?

Reply:
Resolved.